cloud-gcp/src/test/java/com/sequenceiq/cloudbreak/cloud/gcp/setup/GcpProvisionSetupTest.java

```diff
@@ -1,16 +1,17 @@
 package com.sequenceiq.cloudbreak.cloud.gcp.setup;
 
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.Mockito.doNothing;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 import java.io.IOException;
-import java.util.List;
 import java.util.Map;
 
 import org.junit.Assert;
```

Comment:
Should be migrated to its JUnit 5 equivalent (`org.junit.jupiter.api.Assertions` that is already imported above statically).